phpstan-console.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+unset($REX);
+$REX['REDAXO'] = true;
+$REX['HTDOCS_PATH'] = './';
+$REX['BACKEND_FOLDER'] = 'redaxo';
+
+$application = require __DIR__.'/redaxo/src/core/console.php';
+return $application;
```

Comment:
Eigentlich müsste es auch so gehen:

```suggestion
require __DIR__.'/redaxo/src/core/console.php';
return rex::getConsole();
```

Dann müsstest du nicht die console anpassen.